### ISSUE TITLE
Accept integer wavelengths

### ISSUE THREAD

**Issue by abostroem** [kbarbary/extinction#6]
When I make a numpy array of integers, I get a ValueError. It would be nice to be able to use integers as wavelengths. I don't speak Cython - is this an easy thing to implement? 

Here is the call:
extinction.ccm89(np.array([2000]), 1, 3.1)

And the error:
---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
<ipython-input-45-f0f60ea9916f> in <module>()
----> 1 extinction.ccm89(np.array([2000]), 1, 3.1)

extinction.pyx in extinction.ccm89 (extinction.c:3133)()

ValueError: Buffer dtype mismatch, expected 'double' but got 'long'

**Comment by kbarbary** [comment]
Yeah this would be nice. (Also to accept scalars and lists.) When I tried this before it actually added significant overhead for small arrays, but I should check how bad it really was.
